atomics/T1120/T1120.yaml

```diff
@@ -33,3 +33,14 @@ atomic_tests:
     command: |-
       fsutil fsinfo drives
     name: command_prompt
+- name: Get Printer Device List via PowerShell Command
+  description: |
+    This test uses PowerShell to list printers on a Windows system, demonstrating a discovery technique attackers might use to 
+    gather details on connected devices. Using Get-Printer, they can view information on all available printers, identifying 
+    potential devices for further targeting.
+  supported_platforms:
+  - windows
+  executor:
+    name: command_prompt
+    command: |
+      powershell.exe -c "Get-Printer"
```

Comment:
Hello @krdmnbrk Thank you for your contribution. Can you change the executor to `powershell` and command to `Get-Printer`  so that users don't have to create another shell which is not needed here ?